sandbox/plc4py/pyproject.toml

```diff
@@ -24,6 +24,7 @@ authors = ["Apache PLC4X <dev@plc4x.apache.org>"]
 
 [tool.poetry.dependencies]
 python = "^3.8"
+numpy = "^1.19"
 
 [tool.poetry.dev-dependencies]
 pytest = "^5.2"
```

Comment:
Wouldn't it be better to put explicit versions in here and not version ranges? (Or is this more like our prerequisite check in PLC4X where we check at least a given version is installed?)

Reply:
1) Already deleted numpy
2) We will follow PEP-517 https://www.python.org/dev/peps/pep-0517/ and define which minimum version of a module is required for the building.

Reply:
But doesn't this make things not reproducible?